tapis-project/authenticator
# Revert "enhance: WP-308 form footer "Powered by: [tapis]""

## Overview

Revert #60 to fix Jinja bug it causes —

```
{"message":"Unrecognized exception type: <class 'jinja2.exceptions.UndefinedError'>. Exception: there is no parent block called 'head_extra'.","metadata":{},"result":null,"status":"error","version":"dev"}
```

— so Portal team can login to dev.

## Related

- reverts #60
- revisited by #62

## Testing

1. Load project website.
2. Open any redesigned form.
3. Verify form loads.

## UI

| before | after |
| - | - |
| ![error](https://github.com/tapis-project/authenticator/assets/62723358/2c753a6f-c4a4-4510-9161-faa806c358d1) | … |
